DesktopUI2/DesktopUI2/Models/ConfigManager.cs

```diff
@@ -25,12 +25,16 @@ public static Config Load()
       var oldConfig = ConfigStorage.GetObject("config");
       var newConfig = ConfigStorage.GetObject("configDUI");
 
-      if (!string.IsNullOrEmpty(newConfig))
+      Config deserializedConfig = !string.IsNullOrWhiteSpace(newConfig)
+        ? JsonConvert.DeserializeObject<Config>(newConfig)
+        : JsonConvert.DeserializeObject<Config>(oldConfig);
+
+      if (deserializedConfig is not null)
       {
-        return JsonConvert.DeserializeObject<Config>(newConfig);
+        deserializedConfig.UseFe2 = true;
```

Comment:
Am I correct in saying that `UseFe2` will always be true after this.
No more FE1 terminology.

If that's the case, would we be able to kill the `Fe2TextValueConverter` and `Formatting.ReplaceTerminology`

Maybe not now, but we may want to do this eventually. Are there any advantages to keep it?

Reply:
This just sets the intial state of the FE2 terminology button to true - users can still toggle the state if they wish. I believe we decided that leaving this option here for now is okay, if using new terminology is the default.

Reply:
Update to the decision: we'll open a separate ticket to deprecate terminology-switching code

Reply:
Update #2: I've decided to remove everything because it was about the same amount of work amending the previous pr anyways